### PR TITLE
New db connection

### PR DESCRIPTION
**Pull request by Dvogel345**
getting connection to routes setup, need to have my prs up to date

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -27,14 +20,8 @@ function App() {
...
             <Route exact path="/" element={<Main />} />
             <Route path="/comments" element={<Comments />} />
-            <Route path="/search" element={<SearchPage />} />
-            <Route path="/sign-up" element={<Comments />} />
-            <Route path="/movie" element={<MoviesTab />} />
-            <Route path="/show" element={<TvShowsTab />} />
-            <Route path="/developers" element={<DevTeam />} />
-            <Route path="/showtimes" element={<ShowtimesTab />} />
-            <Route path="/dashboard" element={<Dashboard />} />
-            <Route path="/podcast" element={<PodCast/>}/>
```
Comment (omarmoreta) [inline src/App.js:37]: no sure if deleting all of these pages is going to make the app work properly

File: src/App.js [hunk 2]
```diff
@@ -9,14 +9,7 @@ import Footer from "./components/Footer";
...
 import Main from "./pages/Main";
 import Comments from "./pages/Comments";
-import SearchPage from "./pages/Search";
-import MoviesTab from "./pages/MoviesTab";
-import TvShowsTab from "./pages/TvShowsTab";
-import DevTeam from "./pages/DevTeam";
-import ShowtimesTab from "./pages/ShowtimesTab";
-import Dashboard from "./pages/Dashboard";
-import PodCast from "./pages/Podcast";
-
```
Comment (omarmoreta) [inline src/App.js:19]: I think we might need these imports at the app level

**Review by omarmoreta — COMMENTED** [review]
there were a few changes to the app.js, im not sure if the app will function the same without the components that were deleted

File: src/App.js (hunk 2 above)
Comment (Dvogel345) [reply src/App.js:19]: i didnt delete these idk whats up with this

File: src/App.js (hunk 1 above)
Comment (Dvogel345) [reply src/App.js:37]: also for these

**Review by omarmoreta — APPROVED** [review]
Looks good to me

**Pull request merged**